feed.go

```diff
@@ -253,7 +253,7 @@ func (f *baseFeed) GetFollowings(limit int, offset int) ([]FeedID, error) {
 		outputFeeds []FeedID
 	)
 
-	endpoint := "feed/" + f.FeedSlug + "/" + f.UserID + "/" + "following" + "/"
+	endpoint := "feed/" + f.FeedSlug + "/" + f.UserID + "/" + "follows" + "/"
```

Comment:
The docs document only `follows/` so it's better to keep the implementation consistent with the documentation.